src/clients/worker.rs

```diff
@@ -291,59 +307,71 @@ impl<C: golem_cloud_client::api::WorkerClient + Sync + Send> WorkerClient for Wo
-            let message: Message = message.unwrap();
-
-            let msg = match message {
-                Message::Text(str) => {
-                    let parsed: serde_json::Result<InstanceConnectMessage> =
-                        serde_json::from_str(&str);
-                    Some(parsed.unwrap()) // TODO: error handling
-                }
-                Message::Binary(data) => {
-                    let parsed: serde_json::Result<InstanceConnectMessage> =
-                        serde_json::from_slice(&data);
-                    Some(parsed.unwrap()) // TODO: error handling
-                }
-                Message::Ping(_) => {
-                    debug!("Ignore ping");
-                    None
+        let read_res = read.for_each(|message_or_error| async {
+            match message_or_error {
+                Err(error) => {
+                    print!("Error reading message: {}", error);
                 }
-                Message::Pong(_) => {
-                    debug!("Ignore pong");
-                    None
-                }
-                Message::Close(_) => {
-                    info!("Ignore unexpected close");
-                    None
-                }
-                Message::Frame(_) => {
-                    info!("Ignore unexpected frame");
-                    None
-                }
-            };
-
-            match msg {
-                None => {}
-                Some(msg) => match msg.event {
-                    WorkerEvent::Stdout(StdOutLog { message }) => {
-                        print!("{message}")
+                Ok(message) => {
+                    let instance_connect_msg = match message {
+                        Message::Text(str) => {
+                            let parsed: serde_json::Result<InstanceConnectMessage> =
+                                serde_json::from_str(&str);
+                            Some(parsed.unwrap()) // TODO: error handling
+                        }
+                        Message::Binary(data) => {
+                            let parsed: serde_json::Result<InstanceConnectMessage> =
+                                serde_json::from_slice(&data);
+                            Some(parsed.unwrap()) // TODO: error handling
+                        }
+                        Message::Ping(_) => {
+                            debug!("Ignore ping");
+                            None
+                        }
+                        Message::Pong(_) => {
+                            debug!("Ignore pong");
+                            None
+                        }
+                        Message::Close(details) => {
+                            match details {
+                                Some(closed_frame) => {
+                                    print!("Connection Closed: {}", closed_frame);
+                                }
+                                None => {
+                                    print!("Connection Closed");
```

Comment:
`info!`  or `debug!`

Reply:
Used because it was used in other places. I can make that change

Reply:
I think I'm right but please verify with @senia-psm and if needed fix it in both repos

Reply:
Stdout is for results only. In case of `connect` it is only for worker stdout and stderr.

Reply:
Is this connection closed with error message a result?

Reply:
I just want user to know things.

Reply:
If a user passed a wrong worker-id, how should this be printed back to user?

Reply:
```
$ cat src/clients/worker.rs | grep print
                        print!("{message}")
                        print!("{message}")
```
What other places are we talking about?

Where are `print` calls only in 2 places in worker (worker stdout and stderr), 1 place in `auth` (for OAuth2 message) and in `main` for result.

Reply:
We have stderr to inform users of errors. All `info!`, `debug!`, etc are configured to `stderr`. You can use `error!` to show message on default verbosity.

Reply:
Perfect, done